Transcripts/EP_25_Michaela.md

```diff
@@ -13,40 +13,66 @@ came into tech, a little bit (about) my experiences with different companies, an
 will tell you about my favorite topic: code review. So yeah, let's see how that 
 goes. 
 
-[00:00:42] I'm a little bit nervous because it's the first "not-interview-style" podcast, so it means that I don't have another person interviewing me and asking me questions, so it will be all me talking and I hope that this goes well, I don't know. So yeah, about me. Well, should I start? Maybe really from the beginning, like I do with my guests. So how did I actually come into Tech, you know, that's quite a strange story because I went to a high school that was all about art. And while it's way back then a computer were already a thing, I'm not that old, but, uh, at my school, we didn't have laptops in our class. We didn't write on a word for example, or some word process or something.
+<<<<<<< HEAD
```

Comment:
`<<<<<<< HEAD`
This is a marker that indicates that there was a merge conflict. Once the merge conflict is (manually) resolved, this marker should not be there anymore. Resolving a merge conflict can be quite a tricky endeavor, and has to be done with care. Otherwise, the code/text can end up in incorrect state. There are many ways to go about it. Sometimes tool support helps a lot. IDEs such as Pycharm, Visual Studio etc. allow you to resolve merge conflicts.